Add some additional info to gRPC retry section (#737)
temporalio/documentation: docs/typescript/troubleshooting.md

}
```

Several conditions can cause this error, including network hiccups, timeouts that are too short, and an overloaded server.

> Querying a Workflow whose query handler causes an error can result in the query call timing out.

Reviewer: ```suggestion
Querying a Workflow Execution whose query handler causes an error can result in the query call timing out.
```

```

> This happens for a number of reasons: Network hiccups, timeouts that are too short, server overloaded.

Reviewer 2: ```suggestion
Several conditions can cause this error, including network hiccups, timeouts that are too short, and an overloaded server.
```

> Querying a workflow whose query handler causes an error can result in the query call timing out.

Reviewer: Is this troubleshooting content specific to TS?

Author: Nope, possible for anything

> Querying a workflow whose query handler causes an error can result in the query call timing out.

Reviewer 2: ```suggestion
Querying a Workflow whose query handler causes an error can result in the query call timing out.
```

Some troubleshooting actions you can take:

> - Verify the connection from your worker to the Temporal Server is working and doesn't have unusually high latency

Reviewer 2: ```suggestion
- Verify the connection from your Worker to the Temporal Server is working and doesn't have unusually high latency
```

Reviewer: How does one verify this?

- If you are running Temporal Server yourself, check your [server metrics](/docs/server/production-deployment/#scaling-and-metrics) to ensure it's not overloaded

> - If what's timing out is a query, check the logs of your workers to see if they are having issues handling the query

Reviewer 2: ```suggestion
- If what's timing out is a query, check the logs of your Workers to see if they are having issues handling the query
```

Reviewer: How to check Worker logs ? 
Is this the same as looking at the logged output to the process - or is it writing to a file someplace?

> If none of the above actions help you discover why timeouts are occuring, please try to produce a minimal repro and we'll be glad to help.

Reviewer 2: ```suggestion
If none of the preceding actions help you discover why timeouts are occurring, please try to produce a minimal repro and we'll be glad to help.
```
